starter-code/app.js

```diff
@@ -5,6 +5,13 @@ const logger       = require('morgan');
 const cookieParser = require('cookie-parser');
 const bodyParser   = require('body-parser');
 const mongoose     = require("mongoose");
+const session = require('express-session');
+const bcrypt = require('bcrypt');
+const passport = require('passport');
+const LocalStrategy = require('passport-local').Strategy;
+
+const authRoutes = require('./routes/auth-routes');
+const User = require('./models/user');
 
 const app = express();
 
```

Comment:
Replace the mongoose connection (line 22) with the improved one in the cheat sheet.